packages/aws-cdk-lib/aws-dynamodb/lib/table.ts

```diff
@@ -397,6 +397,13 @@ export interface TableOptions extends SchemaOptions {
    * @default - No resource policy statement
    */
   readonly resourcePolicy?: iam.PolicyDocument;
+
+  /**
+   * Resource policy to assign to a DynamoDB stream.
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-globaltable-replicaspecification.html#cfn-dynamodb-globaltable-replicaspecification-replicastreamspecification
```

Comment:
```suggestion
   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-table-streamspecification.html#cfn-dynamodb-table-streamspecification-resourcepolicy
```

since this Table(v1) uses the `AWS::DynamoDB::Table` resource type.

Reply:
Thank you. Copy and paste let me down 😂